package.json

```diff
@@ -93,7 +93,7 @@
     "politespace": "^0.1.4",
     "run-sequence": "^1.1.5",
     "simplecrawler": "^1.1.3",
-    "uswds": "1.4.3",
+    "uswds": "^1.4.4",
```

Comment:
Carrot needs to be removed. We usually add this manually, so maybe that step should be removed.

Reply:
Ok, will edit the line in the release that says to install via the `npm install --save uswds@{{ version }}` command